dietpi-install.sh

```diff
@@ -14,6 +16,125 @@ cleanup() {
 # Trap Ctrl+C and other interrupts
 trap cleanup INT TERM
 
+# Verify SHA256 checksum
+verify_sha256() {
+    local image_file="$1"
+    local checksum_url="$2"
+
+    echo "Downloading SHA256 checksum..."
+    if ! wget -q "$checksum_url" -O "${image_file}.sha256"; then
+        echo "Warning: Could not download checksum file from $checksum_url"
+        return 1
+    fi
+
+    echo "Verifying SHA256 checksum..."
+    if ! sha256sum -c "${image_file}.sha256" 2>/dev/null | grep -q "OK"; then
```

Comment:
Same here, better show error messages for now, which can give some helpful details:
```suggestion
    if ! sha256sum -c "${image_file}.sha256" | grep -q "OK"; then
```